addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -13,6 +13,70 @@ import { pivotTimeAdapter } from "./pivot_time_adapters";
 const { toString, toNumber, toBoolean } = helpers;
 const { DEFAULT_LOCALE } = constants;
 
+class PivotMeasure {
+    constructor(metaData, fieldName) {
+        this._metaData = metaData;
+        this._fieldName = fieldName;
```

Comment:
maybe we can just assign it directly instead of a getter? with `@readonly` if we want ?
```suggestion
        /** @readonly */
        this.name = fieldName;
```

Reply:
Actually, if everything is readonly, we can also assign `display` in the constructor and we don't need to keep `this._metaData`.
M'enfin, ça change pas grand chose